MixItUp.WPF/Controls/MainControls/InteractiveControl.xaml.cs

```diff
@@ -287,7 +287,7 @@ private void NewGameButton_Click(object sender, System.Windows.RoutedEventArgs e
 
         private void MixerLabButton_Click(object sender, RoutedEventArgs e)
         {
-            Process.Start("https://mixer.com/lab/");
+            Process.Start("explorer.exe", "https://mixer.com/lab/");
```

Comment:
This is a bit weird. I know we did it in a few places, but this one was once here.  This use "explorer.exe" to launch the URI, it is supposed to be better supported.  I considered changing throughout the app, but until we see it happen more, I'd hesitant to make that big of a change.

Reply:
Yea, it typically stems from someone not having a default browser set in their windows settings. Apparently it'll sometimes happen with Windows Updates, but I've not been able to track it down to any specific thing. I'll file an issue for us to go through the whole app and fix it, we can make an API in the IFileService to make it easier and just funnel everything through it.

Reply:
https://github.com/SaviorXTanren/mixer-mixitup/issues/229